### PR TITLE
Update the QGIS raster analysis algorithms - table for parameters

### PR DESCRIPTION
**Pull request by havatv**
I found some algorithms that had not been updated to use tables for parameters.

Goal:

Ticket(s): #4429

- [x] Backport to LTR documentation is required

### CODE REVIEW

**Comment by havatv** [comment]
Strange travis error: 

> /opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/sphinx_rtd_theme/search.html:21: RemovedInSphinx30Warning: To modify script_files in the theme is deprecated. Please insert a <script> tag directly in your theme instead.
>   {% endblock %}
> make: *** [html] Error 1
> Makefile:32: recipe for target 'html' failed
> ##[error]Process completed with exit code 2.

File: docs/user_manual/processing_algs/qgis/rasteranalysis.rst [hunk 1]
```diff
@@ -1087,68 +1160,169 @@ of the raster layer that intersects the polygon(s).
...
+     - ``INPUT_VECTOR``
+     - [vector: polygon]
+     - Vector polygon layer that defines the zones.
+   * - **Output column prefix**
+     - ``COLUMN_PREFIX``
+     - [string]
 
-``Vector layer containing the zones`` [vector: polygon]
-  Overlaying vector layer where unique raster values will be appended.
+       Default: ``'HISTO_'``
```
Comment (DelazJ) [inline docs/user_manual/processing_algs/qgis/rasteranalysis.rst:1190]:
```suggestion
       Default: 'HISTO\ _'
```
should work

**Comment by DelazJ** [comment]
The theme error is nothing new, it's from readthedocs and should not prevent the build. There might be rst error elsewhere.

**Comment by havatv** [comment]
> The theme error is nothing new, it's from readthedocs and should not prevent the build. There might be rst error elsewhere.

It could have been the underscore issue?

File: docs/user_manual/processing_algs/qgis/rasteranalysis.rst (hunk 1 above)
Comment (havatv) [reply docs/user_manual/processing_algs/qgis/rasteranalysis.rst]: I applied this (but removed the space) in a separate commit. Thanks!

**Pull request merged**